packages/react-scripts/fixtures/kitchensink/src/App.js

```diff
@@ -1,24 +1,36 @@
-import React from 'react';
+import React, { Component, PropTypes, createElement } from 'react';
 
 class BuiltEmitter extends React.Component {
```

Comment:
You've got `Component` imported now, maybe use it here instead of `React.Component`?